app/params/config.go

```diff
@@ -93,11 +95,9 @@ func SetTendermintConfigs(config *tmcfg.Config) {
 	config.Consensus.GossipTransactionKeyOnly = true
 	config.Consensus.UnsafeProposeTimeoutOverride = 1 * time.Second
```

Comment:
can we change this to 5 seconds? i have a theory that a lower value leads to a lot more rounds (more prevotes of nil) which makes consensus slow when we barely have 67%

Reply:
yessir